service/controller/v18/resource/ipam/create_test.go

```diff
@@ -16,153 +19,380 @@ func mustParseCIDR(val string) net.IPNet {
 	return *n
 }
 
-func Test_canonicalizeSubnets(t *testing.T) {
```

Comment:
Goddammit... This diff exploded... What happened here is that I removed original implementation of `canonicalizeSubnets()` along with its tests as it's now present in `ipam` library.